pshops_lib/registration_check.js

```diff
@@ -0,0 +1,78 @@
+/*
+    Function: registration_check.js
+    Purpose: allows a client to register their SHS product
+*/
+
+// STEP 1 GET A TABLE WHERE SERIAL # IS SAVED
+var table = project.getOrCreateDataTable("SerialNumberTable");
+
+
+// STEP 2 CHECK IF PERSON HAS REGISTERED
+ListRows = table.queryRows({
```

Comment:
let's discuss naming conventions and camelCase vs snake_case vs CapitalizedWords (this is definitely a tom artifact)